packages/sku/config/eslint/eslintConfig.js

```diff
@@ -6,6 +6,11 @@ const baseConfig = {
   extends: orderImports
     ? [coreConfig, require.resolve('./importOrderConfig')]
     : coreConfig,
+  rules: {
+    // Temporarily disable this rule until eslint-config-seek is updated.
+    // This goes hand-in-hand with `compilerOptions.jsx: 'react-jsx'` in tsconfig.json
+    'react/react-in-jsx-scope': 'off',
```

Comment:
1. Release sku
2. Update and release eslint-config-seek
3. Come back here and remove the exception